test/test_distributions.py

```diff
@@ -3014,6 +3014,7 @@ def __init__(self, probs):
             (binomial30, binomial30),
             (binomial_vectorized_count, binomial_vectorized_count),
             (categorical, categorical),
+            (Categorical(torch.ones(1, 10)), Categorical(torch.ones(3, 10))),
```

Comment:
Tuple members in this list are not instances of distributions, but a list of distributions. You could perhaps add a test in `self.infinite_examples` below, and ensure that the KL is indeed infinity.